br/pkg/utils/retry.go

```diff
@@ -24,6 +24,7 @@ var retryableServerError = []string{
 	"error during dispatch",
 	"put object timeout",
 	"internalerror",
+	"not read from or written to within the timeout period",
```

Comment:
I think maybe we should add some retry even we meet the error not in the list.

Reply:
For safety,  add this error into list currenty, I suggest.
But we add the logic(wait and retry the operation(backup and retry)) into master branch until the work is stable and safe.